spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -919,6 +933,8 @@ public IntegrationFlow scatterGatherAndExecutorChannelSubFlow(TaskExecutor taskE
 					.build();
 		}
 
+
+
```

Comment:
This like is redundant.

Reply:
No changes performed by me here.

Reply:
Well, GitHub thinks differently and shows me two blank lines.
Only one has to be in between.

Well, don't worry: I clean up the code style on merge: just wanted to let you know that we try to follow some code style. We develop here an Open Source, so our code has to be perfect for community readers.
Therefore, if you are going to follow our rules, we are not going to have discussions like this any more and that will save our time on review and merging process.

Thanks for understanding.

Reply:
I was under impression grade build throws all issues with code style. But it's not throwing these issues. Next time will ensure manual verification of these

Reply:
Yeah... The Checkstyle tool is just a tool written by human. Since not all tasks human can perform, it is not a surprise that their tools are not perfect.
More over how can the tool be sure that these two lines is not your intention.
Therefore my question about them: with a good reason we can leave them.
But for great code it would be better to avoid unnecessary changes.